winui/Masked-TextBox/Events.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: Events in WinUI MaskedTextBox control | Syncfusion
+description: Learn all about the events in the MaskedTextBox (SfMaskedTextBox) control.
+platform: WinUI
+control: SfMaskedTextBox
+documentation: ug
+---
+
+# Events in WinUI MaskedTextBox
+
+This section explains about the events exposed by the MaskedTextBox control.
+
```

Comment:
@BhaskarSuresh22 This section explains the events available in the WinUI MaskedTextBox control.

Reply:
@Manivannan-E  Done the changes